host/xtest/regression_1000.c

```diff
@@ -2570,11 +2570,14 @@ static void xtest_tee_test_1034(ADBG_Case_t *c)
 
 	res = xtest_teec_open_session(&session, &large_ta_uuid, NULL,
 				      &ret_orig);
+
 	if (res == TEEC_ERROR_OUT_OF_MEMORY) {
 		Do_ADBG_Log("TEEC_ERROR_OUT_OF_MEMORY - ignored");
+		return;
```

Comment:
This isn't needed since we have the `else` below. Alternatively, we could keep this `return`, but then we don't need the `else`.

Reply:
Thanks for your review. According the log, the ADBG_EXPECT_TEEC_SUCCESS(c, res) function response with TEEC_ERROR_ITEM_NOT_FOUND this error message. Did you means remove all else condition as below:

else {
		ADBG_EXPECT_TEEC_SUCCESS(c, res);
		if (!res)
			TEEC_CloseSession(&session);
}

If we remove all else condition that means we will not see TEEC_ERROR_ITEM_NOT_FOUND this error message. Thus, I think this change is else condition is good enough that provided by @jforissier 
@jforissier Could you help us to review and merge it if this commit is valuable? Thanks.

Sincerely,
Tim

Reply:
Jens is right, what is not needed is the `else` keyword, not the `else` block. You did not copy the code I suggested in #682 properly:
```C
	if (res == TEEC_ERROR_OUT_OF_MEMORY) {
		Do_ADBG_Log("TEEC_ERROR_OUT_OF_MEMORY - ignored");
		return;
	}
	ADBG_EXPECT_TEEC_SUCCESS(c, res);
	if (!res)
		TEEC_CloseSession(&session);
```

The other valid option is:
```C
	if (res == TEEC_ERROR_OUT_OF_MEMORY) {
		Do_ADBG_Log("TEEC_ERROR_OUT_OF_MEMORY - ignored");
	} else {
		ADBG_EXPECT_TEEC_SUCCESS(c, res);
		if (!res)
			TEEC_CloseSession(&session);
	}
```
So please pick one of the two.

Reply:
Thanks @jforissier, Sorry about that, I make mistake that add extra "else". I will correct it. Thanks for your correction. :)